app/src/main/java/io/github/pulquero/rotordroid/Node.java

```diff
@@ -75,6 +75,9 @@ public LapStats readLapStats(long currentTime) throws IOException {
         byte laps = buf[0];
         int msSinceLastLap = read16(buf, 1);
         stats.rssi = buf[3];
+        if (stats.rssi < 0) {
```

Comment:
Not quite sure what this is about. Note rssi < 0 (unsigned 0xFF) is used to indicate an error. Makes better sense to throw an exception or skip the reading.